source/common/router/router.cc

```diff
@@ -648,7 +648,7 @@ void Filter::onUpstreamReset(Http::StreamResetReason reset_reason) {
   }
 
   const StreamInfo::ResponseFlag response_flags = streamResetReasonToResponseFlag(reset_reason);
-  const std::string body =
+  const absl::string_view body =
```

Comment:
This one have to be string otherwise you'll have dangling string_view, you can pass string as string_view